### PR TITLE
fix: tests for age filter and similar events

### PR DESCRIPTION
**Pull request by melniiv**
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr160.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr160.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr160.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr160.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr160.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr160.dev.hel.ninja :laughing::tada::tada::tada:

File: apps/hobbies-helsinki/src/domain/event/queryUtils.ts [hunk 1]
```diff
@@ -31,18 +31,21 @@ const useSimilarEventsQueryVariables = (event: EventFields) => {
         .map((category) => category?.id) // collect ids
         .filter((id) => id != null) // remove nulls and undefined ones
         .join(), // make a string
-      [EVENT_SEARCH_FILTERS.MIN_AGE]: event.audienceMinAge ?? '',
-      [EVENT_SEARCH_FILTERS.MAX_AGE]: event.audienceMaxAge ?? '',
+      [EVENT_SEARCH_FILTERS.MIN_AGE]: event.audienceMinAge ?? '5',
+      [EVENT_SEARCH_FILTERS.MAX_AGE]: event.audienceMaxAge ?? '15',
```
Comment (karisal-anders) [inline apps/hobbies-helsinki/src/domain/event/queryUtils.ts:35]: Is the default age limit 5–15 here intended for tests only? I mean `useSimilarEventsQueryVariables` -> `useSimilarEventsQuery` -> `SimilarEvents` -> `EventPageContainer` which is used in other than tests so this affects not only tests.

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr160.dev.hel.ninja** :rocket::rocket::rocket:

**Review by nikomakela — APPROVED** [review]
The tests seems to work. I don't fully understand the changes that are done. Was the problem really in the actual queries and not in the test mocks? What I was waiting from this PR was some changes to the mocks.

File: apps/hobbies-helsinki/src/domain/event/queryUtils.ts (hunk 1 above)
Comment (nikomakela) [reply apps/hobbies-helsinki/src/domain/event/queryUtils.ts]: oh yea. I want to remove my approval! :D

**Review by nikomakela — CHANGES_REQUESTED** [review]
The ages should not be hard coded to the query. Instead, the test mocks should be fixed.

**Review by nikomakela — APPROVED** [review]

**Pull request merged**